scss/components/_program.scss

```diff
@@ -21,3 +20,10 @@
     text-decoration: underline;
   }
 }
+
+@media screen and (max-width: 767px) {
+  .icons {
+    font-size: 1.5rem;
+    margin-right: 0.4rem !important;
+  }
+}
```

Comment:
- [ ] Kindly use a mobile-first approach. It means the default styling should be for mobile view and the media quary should be applied to a bigger screen (desktop). Kindly apply this to other scss files with the same issue.